docs/examples.md

```diff
@@ -1183,6 +1183,12 @@ overrides:
           y: 100
 ```
 
+## Doorbell
```

Comment:
```suggestion
### Doorbell
```

Reply:
It was actually a mistake, please check again.